Angular 13 | Cannot find name 'IDBArrayKey'

After upgrading to Angular 13 I receive the following error when I try to build/serve my application

```bash
Error: node_modules/ngx-indexed-db/lib/ngx-indexed-db.meta.d.ts:41:84 - error TS2304: Cannot find name 'IDBArrayKey'.

export declare type Key = string | number | Date | ArrayBufferView | ArrayBuffer | IDBArrayKey | IDBKeyRange;
```

These are the dependencies I am using:

```json
"@angular/*": "13.*.*",
// …
"ngx-indexed-db": "9.4.0",
// …
"typescript": "4.4.4",
```

--- a/projects/ngx-indexed-db/src/lib/ngx-indexed-db.meta.ts
+++ b/projects/ngx-indexed-db/src/lib/ngx-indexed-db.meta.ts
@@ -37,6 +37,6 @@ export enum DBMode {
   readwrite = 'readwrite',
 }
 
-export type Key = string | number | Date | ArrayBufferView | ArrayBuffer | IDBArrayKey | IDBKeyRange;
+export type Key = string | number | Date | ArrayBufferView | ArrayBuffer | IDBValidKey | IDBKeyRange;
 
 export const CONFIG_TOKEN = new InjectionToken<DBConfig>(null);

